### DISCUSSION TITLE
document.onSnapshot is not a function when trying to sync a single value instead of a collection

### DISCUSSION THREAD

**Discussion by holmesadrianh** [vuejs/vuefire#1303] [Questions and Help] [answered]
### Reproduction

this.$bind( "isAdmin", db.collection('users').doc(userId).get("isAdmin"))

### Steps to reproduce the bug

this.$bind( "isAdmin", db.collection('users').doc(userId).get("isAdmin"))

### Expected behavior

I expect the single boolean value in firebase to sync in realtime with the boolean value in Vue.

### Actual behavior

I get an error:

Uncaught (in promise) TypeError: document.onSnapshot is not a function

### Additional information

I did not see anything in the documentation for VueFire 2 about realtime individuals values instead of collections.

**Comment by posva** [comment, marked as answer]
the `get()` function already returns a promise. VueFire can only bind documents or collections, it cannot bind a single value as there wouldn't been automatic updates, which is where the `onSnapshot()` is not a function comes from 🙂 

BTW, you should definitely use Firebase v9 + VueFire v3. The v2 is very old and is not receiving any updates anymore.
